### PR TITLE
Replace "Raleigh Design" with "Triangle Designers"

### PR DESCRIPTION
**Pull request by creuter**
We've built a robust triangle-wide Slack community, and are going to
repurpose this application for triangledesigners.com.

The first step is to change all of the Raleigh Design references.

### CODE REVIEW

File: web/email.ex [hunk 1]
```diff
@@ -5,39 +5,39 @@ defmodule Community.Email do
   def admin_job_posted(job) do
     new_email
     |> to(admin_emails)
-    |> from("noreply@raleighdesign.io")
+    |> from("noreply@triangledesigners.com")
```
Comment (drapergeek) [inline web/email.ex]: What do you think about extracting an attribute here: `@reply_email` so we can avoid repeating so much?

**Comment by drapergeek** [comment]
Do you think you should _somewhere_ reference "which" Triangle this is referring to?

**Comment by creuter** [comment]
@drapergeek Somehow my ag missed a couple "Raleigh"s, which I updated.

The home page description prominently says 

> A resource for designers in throughout North Carolina’s Triangle to stay connected, learn about upcoming events and find prospective career opportunities.

File: web/email.ex [hunk 2]
```diff
@@ -5,46 +5,50 @@ defmodule Community.Email do
...
     |> render("contact_form.text", body: data.body, from: data.from)
   end
 
   defp admin_emails do
     System.get_env("ADMIN_EMAILS")
     |> String.split(",")
   end
+
+  def noreply_email do
+    System.get_env("NOREPLY_EMAIL")
```
Comment (drapergeek) [inline web/email.ex]: Even better!


Mind changing this to `defp`?

**Comment by drapergeek** [comment]
1 comment, :shipit:

**Pull request closed without merging**